pkg/standalone-utils/contracts/ProtocolFeeSweeper.sol

```diff
@@ -80,7 +80,7 @@ contract ProtocolFeeSweeper is IProtocolFeeSweeper, SingletonAuthentication, Ree
                     _transferFeeToken(pool, feeToken, withdrawnBalance);
                 } else {
                     // Transfer the tokens directly to avoid "hanging approvals," in case the burn is unsuccessful.
```

Comment:
```suggestion
                    // We must revert if this allowance is not entirely consumed by the burner, to avoid exploitable
                    // "hanging approvals." The order may be asynchronous, but it must pull the tokens immediately.
```